shared/constants/flask/environment.js

```diff
@@ -0,0 +1,2 @@
+export const isMain = process.env.METAMASK_BUILD_TYPE === 'main';
```

Comment:
Should this be in a `flask` folder? I think it would be fine to just keep it in `shared/constants/environment.js`